python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -91,6 +93,60 @@ def _build_fil_classifier(m, data, arg={}, tmpdir=None):
                   storage_type=arg["storage_type"])
 
 
+def _build_fil_skl_classifier(m, data, arg={}, tmpdir=None):
+    """Setup function for FIL SKLearn classification benchmarking"""
```

Comment:
Maybe a more descriptive comment, like: `Trains a scikit-learn classifier and returns a FIL-format model version of it`

Reply:
Done.